recipe/meta.yaml

```diff
@@ -27,42 +27,97 @@ source:
     - nvcc.profile.patch.win  # [win]
 
 build:
-  number: 0
+  number: 1
```

Comment:
Since we are adding `1` in PR ( https://github.com/conda-forge/cuda-nvcc-feedstock/pull/5 ), would bump this to `2`

```suggestion
  number: 2
```

Reply:
Nvm looks like you already did 😂